src/nullLogger.ts

```diff
@@ -1,13 +1,15 @@
+import { ILogger, ILoggingConfiguration } from './chrome/internal/services/logging';
```

Comment:
Watch out for the imports above the copyright, autoimport will do that...

Reply:
Fixed